### PR TITLE
release: 2.60.1

### PR DESCRIPTION
**Pull request by mvo5**

### CODE REVIEW

File: packaging/ubuntu-16.04/changelog [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+
+  * New upstream release, LP: #2024007
+    - install: fallback to lazy unmount() in writeFilesystemContent
+    - data: include "modprobe.d" and "modules-load.d" in preseeded blob
+    - gadget: fix install test on armhf
+    - interfaces: fix typo in network_manager_observe
+    - sandbox/apparmor: don't let vendored apparmor conflict with system
+    - gadget/update: set parts in laid out data from the ones matched
+    - many: move SnapConfineAppArmorDir from dirs to sandbox/apparmor
+    - many: stop using `-O no-expr-simplify` in apparmor_parser
```
Comment (pedronis) [inline packaging/ubuntu-16.04/changelog:11]: should this mention that is a stability vs speed trade-off atm?

**Review by pedronis — APPROVED** [review]
thanks

File: packaging/ubuntu-16.04/changelog (hunk 1 above)
Comment (mvo5) [reply packaging/ubuntu-16.04/changelog:11]: Good point, let me add it

Comment (mvo5) [reply packaging/ubuntu-16.04/changelog:11]: I updated NEWS.md to include a more human friendly summary of the changes and their impact

**Review by pedronis — APPROVED** [review]
thanks

**Pull request merged**